### ISSUE TITLE
Locale support?

### ISSUE THREAD

**Issue by 3v1n0** [blueanvil/kotlin-duration-string#1]
Very nice lib, what about locale support (so that it can parse other languages)?

**Comment by cosmin-marginean** [comment]
@3v1n0 Thanks for your feedback! And for this request. 
I belive in theory it's a good idea, but the library design is very crude and inflexible at this time to support something like this. At the same time we could probably achieve this with a simple translation table if it's purely a matter of translating `m`, `s`,`ms` etc to other symbols, but it would be a matter of sourcing relevant (and recognised) translation tables, which could be labourios for what seems like an edge case, at this stage.

In other words, we'd welcome a PR contribution in this area, but we probably won't prioritise this work ourselves any time soon.
